lms/djangoapps/courseware/masquerade.py

```diff
@@ -185,6 +190,25 @@ def get_masquerading_user_group(course_key, user, user_partition):
     # The user is masquerading as a generic student or not masquerading as a group return None
     return None
 
+def check_content_start_date_for_masquerade_user(course_key, user, request, course_start,
```

Comment:
Nit: `check_*` is a bit of a fuzzy name. Looking at the call site, I wasn't sure what was going to happen as part of this method. Maybe something with `warn` in the name? I'm having trouble not making it a mouthful, though.